feature/profile/src/commonMain/kotlin/io/github/confsched/profile/ProfileEditScreen.kt

```diff
@@ -126,11 +128,14 @@ fun ProfileEditScreen(
     val scrollBehavior = TopAppBarDefaults.pinnedScrollBehavior()
     val focusManager = LocalFocusManager.current
 
+    BackHandler(enabled = true, onBack = onBackClick)
+
     Scaffold(
         topBar = {
             AnimatedTextTopAppBar(
                 title = stringResource(ProfileRes.string.profile_card_title),
                 scrollBehavior = scrollBehavior,
+                onBackClick = onBackClick,
```

Comment:
I think this change affects the UI design more than we intended.

What we were really aiming for is just handling the back input when ProfileEditScreen is opened with a completed profile. Could you remove this line to reflect that? 🙏🏻

Reply:
Fixed this in the following commit 🙏 

[cef4c30](https://github.com/DroidKaigi/conference-app-2025/pull/588/commits/cef4c30986edb03b0d84478f8a49a46f365ee143)